src/components/EditorAndOutput.vue

```diff
@@ -170,16 +167,19 @@ const graphsDropdown = computed<SelectMixedOption[]>(() => {
       (fileName): SelectMixedOption => ({
         label: fileName.substring(
           0,
-          fileName.valueOf().length - ".graph".length,
+          fileName.valueOf().length - ".graph".length
         ),
         value: fileName,
-      }),
+      })
     );
 });
 
+type EditorType = "shader" | "graph";
+
 function useOpenFile(startFile: FilePath | null, fs: ReactiveFilesystem) {
   const openedFileName = ref<FilePath | null>(startFile);
   const keyedCode = ref<KeyedCode | null>(null);
+  const editorType = ref<EditorType>("shader");
```

Comment:
Here we make the editor type dependent on the opened file. It is a `ref` instead of a computed, because it should be possible to open a `.graph` file with Monaco as well. 
(For debugging purposes)